system/manager/process_config.py

```diff
@@ -59,7 +59,7 @@ def only_offroad(started, params, CP: car.CarParams) -> bool:
   NativeProcess("sensord", "system/sensord", ["./sensord"], only_onroad, enabled=not PC),
   NativeProcess("ui", "selfdrive/ui", ["./ui"], always_run, watchdog_max_dt=(5 if not PC else None)),
   PythonProcess("soundd", "selfdrive.ui.soundd", only_onroad),
-  NativeProcess("locationd", "selfdrive/locationd", ["./locationd"], only_onroad),
+  PythonProcess("locationd", "selfdrive.locationd.locationd", only_onroad),
```

Comment:
@deanlee this new process uses 2-3x more CPU than the previous locationd. want to see where all that's going?

Reply:
I'm investigating the bottleneck but haven't pinpointed it yet. Any insights or suggestions?

Reply:
It looks like the bottleneck is in `rednose/helpers/ekf_sym_pyx.pyx`